Dockerfile

```diff
@@ -0,0 +1,75 @@
+# -------------------------- Dev ---------------------------------------
+
+FROM node:22-bullseye AS dev
+
+RUN apt-get update -y \
+    && apt-get install -y --no-install-recommends git \
+    && rm -rf /var/lib/apt/lists/* \
+    # NOTE: yarn > 1.22.19 breaks yarn-install invoked by yarn
+    && npm install -g yarn@8.6.0 yarn@1.22.19 --force \
+    && git config --global --add safe.directory /code
+
+WORKDIR /code
+
+# Build stage for web app
+FROM dev AS web-app-serve-build
+
+COPY ./package.json ./yarn.lock /code/
+
+RUN yarn install
+COPY . /code/
+
+ENV VITE_FIREBASE_API_KEY=12
+ENV VITE_FIREBASE_AUTH_DOMAIN=1234
+ENV VITE_FIREBASE_DATABASE_URL=https://mapswipe.org
+ENV VITE_FIREBASE_PROJECT_ID=1235
+ENV VITE_FIREBASE_STORAGE_BUCKET=1245
+ENV VITE_FIREBASE_MESSAGING_SENDER_ID=124
+ENV VITE_FIREBASE_APP_ID=134
```

Comment:
Please make sure that the example values have correct types, for eg:
`VITE_FIREBASE_API_KEY` should match type (and also structure if possible) string.
```
VITE_FIREBASE_API_KEY=example-FIrebaseAP1k3y
```